Description: Exclude buildtest python package from Debian package.
 By default tahoe-lafs also install a buildtest package but this isn't relevant
 in the context of the Debian package.
Forwarded: not-needed
Author: bertagaz <bertagaz@ptitcanardnoir.org>
Index: tahoe.git/setup.py
===================================================================
--- tahoe.git.orig/setup.py	2013-09-03 19:04:12.745099226 +0200
+++ tahoe.git/setup.py	2013-09-03 19:04:24.637106872 +0200
@@ -436,8 +436,7 @@
                 'allmydata.web',
                 'allmydata.web.static',
                 'allmydata.web.static.css',
-                'allmydata.windows',
-                'buildtest'],
+                'allmydata.windows'],
       classifiers=trove_classifiers,
       test_suite="allmydata.test",
       install_requires=install_requires,
Index: tahoe.git/src/allmydata_tahoe.egg-info/SOURCES.txt
===================================================================
--- tahoe.git.orig/src/allmydata_tahoe.egg-info/SOURCES.txt	2013-09-03 19:00:55.393107907 +0200
+++ tahoe.git/src/allmydata_tahoe.egg-info/SOURCES.txt	2013-09-03 19:04:12.753100409 +0200
@@ -422,7 +422,5 @@
 src/allmydata_tahoe.egg-info/not-zip-safe
 src/allmydata_tahoe.egg-info/requires.txt
 src/allmydata_tahoe.egg-info/top_level.txt
-src/buildtest/__init__.py
-src/buildtest/test_build_with_fake_dist.py
 static/tahoe.py
-twisted/plugins/allmydata_trial.py
\ No newline at end of file
+twisted/plugins/allmydata_trial.py
Index: tahoe.git/src/allmydata_tahoe.egg-info/top_level.txt
===================================================================
--- tahoe.git.orig/src/allmydata_tahoe.egg-info/top_level.txt	2013-09-03 19:00:55.393107907 +0200
+++ tahoe.git/src/allmydata_tahoe.egg-info/top_level.txt	2013-09-03 19:04:12.753100409 +0200
@@ -1,2 +1 @@
 allmydata
-buildtest
